deno/lib/__tests__/object.test.ts

```diff
@@ -164,6 +164,7 @@ test("test that optional keys are unset", async () => {
     id: "asdf",
     set: undefined,
   });
+  // eslint-disable-next-line ban/ban
   expect(Object.keys(result)).toEqual(["id", "set"]);
```

Comment:
Do ponyfills not work in Deno?

Reply:
they do - it's a test code, no need to use ponyfills here